tests/TestSuite/Mutant/ResultTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/**
+*
+* @category   Humbug
+* @package    Humbug
+* @copyright  Copyright (c) 2015 Pádraic Brady (http://blog.astrumfutura.com)
+* @license    https://github.com/padraic/humbug/blob/master/LICENSE New BSD License
+* @author     Thibaud Fabre
+*/
+namespace Humbug\Test\TestSuite\Mutant;
+
+use Humbug\TestSuite\Mutant\Result;
+
+class ResultTest extends \PHPUnit_Framework_TestCase
+{
+    public function testGetStatusCodeReturnsCorrectCode()
+    {
+        $this->assertEquals(Result::TIMEOUT, Result::getStatusCode(true, true, true));
+        $this->assertEquals(Result::TIMEOUT, Result::getStatusCode(false, false, true));
+        $this->assertEquals(Result::TIMEOUT, Result::getStatusCode(true, false, true));
```

Comment:
Leaving as is for now but keeping the issue in the back of my head. Removing the Result class is a bit too extreme, and I still don't think adding the methods to collector will solve the issue. No matter how I put it for now, we have the status information (a scalar value) that results from three sources of data (stdout, exit code, and timeout status). Adding the methods to the collector only moves the responsibility of deciding the type of result to whoever calls the methods on the Collector (in current state, the Runner), driving up the complexity of whatever method needs to push a result into the collector. But I may have just had an idea that makes it better :)